database/migrations/2024_11_17_202201_create_letters_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('letters', function (Blueprint $table) {
```

Comment:
Sería viable agregar un campo a mayores the imagen que simplemente contenga una url(string)?